src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java

```diff
@@ -9,31 +9,31 @@
 // xdoc section -- start
 class Test { // violation, class only has a static method and a constructor
 
-  public Test() {
-  }
+    public Test() {
```

Comment:
Please do not close PRs.
Please lean how to reuse PRs.

Indentation should be +2

Reply:
PR closed automatically, I think it happened when I force pushed to fork's master.
Fixed identation

Reply:
@andreiyusupau always allow reviewers to resolve conversations